extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -53,10 +53,10 @@
    z-index: 2147483646;
    width: 100%;
    position: fixed;
-   top: 0;
+   background-color: white;
  }
 
- .i-amphtml-toolbar > ul {
+ .i-amphtml-toolbar > ul:first-child {
```

Comment:
`i-amphtml-toolbar` is the class applied to the `<nav toolbar>` element. I noticed when implementing the sidebarOnly amp start, that these styles we being applied to the ul within the amp-accordian. So I ensured that this was the first child, as <nav toolbar> may only contain one direct child of `<ul>`